### PR TITLE
created the delete button for chapters

### PR DESCRIPTION
**Pull request by ricardojusto**

### CODE REVIEW

File: app/views/backend/chapter.erb [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <div class="col-md-9">
           <div class="row">
             <div class="col-md-10">
-              <h1><%= @chapter.name %></h1>
+              <h1><%= @chapter.name %></h1><br>
```
Comment (rumama) [inline app/views/backend/chapter.erb]: Unneeded <br> element!

File: app/views/backend/course.erb [hunk 2]
```diff
@@ -15,6 +15,7 @@
                 <% @course.chapters.each do |chapter| %>
                    <li>
                     <%= chapter.name %>
+                    <a href="/chapter/delete/<%= chapter.id %>" <i class="material-icons">delete</i></a><br>
```
Comment (rumama) [inline app/views/backend/course.erb]: Unneeded break line element!
The chapter.name && chapter.description should be in separated li tags.

File: app/views/layout.erb [hunk 3]
```diff
@@ -15,6 +15,7 @@
 		<!-- Jquery -->
 		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js">
```
Comment (rumama) [inline app/views/layout.erb]: Please remove this line! 
Jquery is already being used in our project! Check bottom of this file.

File: app/views/layout.erb [hunk 4]
```diff
@@ -15,6 +15,7 @@
...
 		</script>
+		<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (rumama) [inline app/views/layout.erb]: We need to decide which icons we are going to use in our project! Bootstraps or Materialize icons!
We already have bootstrap icons(glyphicons) in some parts of our code. Let's choose one of them and stick with it!

**Comment by rumama** [comment]
:+1:

File: app/views/backend/course.erb (hunk 2 above)
Comment (ricardojusto) [inline app/views/backend/course.erb]: but the <ul> shouldn't be about chapters? if we put each element inside a <li> when we have a course with 10 chapters we won't be able to tell wich chapter name or description belongs to each chapter.

File: app/views/layout.erb (hunk 3 above)
Comment (ricardojusto) [inline app/views/layout.erb]: if that link is removed the add chapter funciontality won't work.

**Comment by ricardojusto** [comment]
opened issue #49 about where the delete button should be

File: app/views/backend/course.erb (hunk 2 above)
Comment (rumama) [inline app/views/backend/course.erb]: Check nested lists! One List to list the chapters name and a nested list inside each chapter to set the description.  One solution could be:
´´´html
<ul>
  <li>Chapter name</li>
    <ul>
    <li>Description</li>
    </ul>
  </li>
  <li>Chapter name</li>
</ul>
´´´
Or use divs instead!

File: app/views/layout.erb (hunk 3 above)
Comment (rumama) [inline app/views/layout.erb]: Then it's a bug! We need to figure it out!

**Comment by rumama** [comment]
:+1:

**Comment by rumama** [comment]
fixed #50

**Pull request merged**